MetaData/tuples/MiniAOD-13TeV_Data.json

```diff
@@ -22,7 +22,11 @@
   "data_DoubleMuon_Run2015D_PromptReco-v4_25ns" : "/DoubleMuon/Run2015D-PromptReco-v4/MINIAOD",
   "data_MuonEG_Run2015D_PromptReco-v4_25ns" : "/MuonEG/Run2015D-PromptReco-v4/MINIAOD",
   "data_DoubleEG_Run2015D_PromptReco-v4_25ns" : "/DoubleEG/Run2015D-PromptReco-v4/MINIAOD",
-  "data_Tau_Run2015D_PromptReco-v4_25ns" : "/Tau/Run2015D-PromptReco-v4/MINIAOD"
+  "data_Tau_Run2015D_PromptReco-v4_25ns" : "/Tau/Run2015D-PromptReco-v4/MINIAOD",
+
+  "data_SingleMuon_Run2015C_PromptReco_25ns" : "/SingleMuon/Run2015C-PromptReco-v1/MINIAOD",
+  "data_SingleMuon_Run2015D_PromptReco-v3_25ns" : "/SingleMuon/Run2015D-PromptReco-v3/MINIAOD",
```

Comment:
These are wrong. Look above to see the correct MINIAOD versions. You can safely copy the DoubleMuon ones and create a new line, double checking the correct v1 in DAS (see 05Oct2015 MuonEG which is v2).